### ISSUE TITLE
Grading/Feedback

### ISSUE THREAD

**Issue by barryross** [milesnextdoor/cmmontes2.github.io#1]
# FEWD Final Project

---
### Description

For the final project, you'll be designing and building a website of your choice. This project will test your knowledge of front-end web development and ask you to apply everything you've learned in this course. The result will be a website that you can add to your portfolio.  You could create: a portfolio website; a marketing website for a startup or business; or a prototype for a simple web-app. Work with your instructor and EIR's to create project goals that are realistic given the scope and timing of the class.

Student: Chris Montes

Project URL: cmmontes2.github.io

|  |  |  |  |
| --- | --- | --- | --- |
| Technical Requirements | Does Not Meet Expectations (0) | Meets Expectations (1) | Exceeds Expectations (2) |
| Project Proposal / Wireframes turned in week 5 |  |  | x |
| Draft of HTML/CSS turned in week 7 | x |  |  |
| First draft of JS turned in week 8 | x |  |  |
| Updated rough draft of project week 9 |  |  | x |
| Final presentation |  |  | x |
| Use HTML5 structural elements (header, footer, nav, footer) |  |  | x |
| Demonstrate a correct use of classes and IDs |  |  | x |
| Use classes to identify a type of element and IDs for identifying a specific unique element on the page |  |  | x |
| Select the appropriate tags to markup content |  |  | x |
| Apply fonts, color and styles to elements and the page |  |  | x |
| Demonstrate use of floats (or inline-block) and the box model |  |  | x |
| Use JavaScript/jQuery to make pages interactive |  |  | x |
| TOTAL: 20/24 |  |  |  |

**Comment by barryross** [comment]
Hey Chris, really great job with the final project and with your presentation!  

I really enjoyed your presentation, as well as reviewing your code and seeing the progress you have made over the course.  I know you have put a lot of time and energy into this and it has been a pleasure having you in class, and watching your work pay off.

I know we spoke about this previously, but I have a couple questions as I am reviewing my notes on homework submissions and milestones for the final project, and trying to ensure accurate numbers.  

For assignments, it looks like I did not receive the following: 
#4 - jQuery Relaxr (actually for this one, I am seeing it in your repo...did you hand this in and I just never graded it for you?  If so, I'll get back to you on it asap )
#5 - CitiPlex pictures
#6 CitiPlex Dropdown

For the milestones, I believe it was #2 (rough draft of html/css) and #3 (draft with JS) that were not met.  Is that correct?  Can you please let me know so I can make a note? 

Thank you!
Barry

**Comment by milesnextdoor** [comment]
Yes, I did hand in the 4th one and just never received the final grade on
it.

#5 i believe i did complete it and i think i may have just forgot to push
it to Git.

#6 I chose to forego as i was running out of time to complete the final
project.

Chris Montes
Cmmontes2@gmail.com
LinkedIn https://www.linkedin.com/in/cmmontes | Twitter
https://twitter.com/cmmontes2 | Facebook
https://www.facebook.com/cmmontes2

**Comment by barryross** [comment]
Okay, thanks for letting me know.  I’m sorry I never got back to you about #4!  I’ll mark it today and get back to you on it.  Also, if you would like to push #5 to Git, I’m happy to review that as well for you.  I’ll update the records with your notes.  

Can you please confirm if I have the Milestones for the final project correct…is that what we spoke about previously?  Sorry to have to ask you again. 

thanks,
Barry

**Comment by milesnextdoor** [comment]
yes the final project milestones are correct. Thanks Barry!

Chris Montes
Cmmontes2@gmail.com
LinkedIn https://www.linkedin.com/in/cmmontes | Twitter
https://twitter.com/cmmontes2 | Facebook
https://www.facebook.com/cmmontes2

**Comment by barryross** [comment]
Great, thank you Chris and you’re welcome!  If you do upload the CitiPlex assignment and want me to have a look, please let me know.  

Thanks,
Barry
